code/style.css

```diff
@@ -1,7 +1,347 @@
+/* styling for the whole page */
+* {margin:0;}
+
+body {
+  background-color: #271916;
+}
+
+/* styling for the header */
+#hero {
+  height: 1000px; 
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#hero video {
+  width: 100%;
+  height: 1000px;
+  position: absolute;
+  top: 0;
+  left:0;
+  z-index: 5;
+  object-fit: cover;
+}
+
+#hero .caption {
+  position: relative;
+  z-index: 20;
```

Comment:
Is there a reason that you put such a big value for the z-index?

Reply:
Nope, no reason, just the lazy way to ensure that the element is always on top :)